src/mlpack/methods/lsh/lsh_search_impl.hpp

```diff
@@ -477,6 +933,23 @@ void LSHSearch<SortPolicy>::Search(const arma::mat& querySet,
   if (k == 0)
     return;
 
+  // If the user requested more than the available number of additional probing
+  // bins, set Teffective to maximum T. Maximum T is 2^numProj - 1
+  size_t Teffective = T;
+  if (T > ( (size_t) ( (1 << numProj) - 1) ) )
+  {
+    Teffective = ( 1 << numProj ) - 1;
```

Comment:
There are a bunch of extra spaces in these three lines:

```
if (T > ((size_t) ((1 << numProj) - 1)))
{
  Teffective = (1 << numProj) - 1;
```

Technically variables should have names that start with lowercase letters, so I would have gone with `t` and `effectiveT` instead.